src/test/java/Hmwk21.java

```diff
@@ -1,2 +1,74 @@
-public class Hmwk21 {
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.annotations.Test;
+
+import java.util.concurrent.TimeUnit;
+
+
+//Using Action Class methods, create/refactor the test case renamePlaylist()
+//        Implement Explicit Waits
+//        Remove all Thread.sleep implementations
+//        Create a new branch and commit your changes
+//        Push your code to a remote repository
+//        Create a pull request
+//        Copy and paste the link of the pull request (or your branch) to the field below, so we can check your homework
```

Comment:
**I recommend not using code comments:** for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.

Reply:
Yeah, I keep doing that and forgetting to take it out. My bad.